routes/barcode.js

```diff
@@ -19,14 +19,15 @@ router.get('/', async (req, res, next) => {
     const images = await article.getImagesFromDateRange(dateFrom, dateTo);
     const config = createConfig(orientation, fit, images.length, width, height);
     const updatedImages = createImages(images, fit, config.width, config.height);
-    const outputPath = './downloads/_result/output.jpg';
+    const downloadPath = process.env.DOWNLOAD_PATH;
```

Comment:
I would use the `path` module to ensure your paths are all relative to the root

Reply:
Done! :)